python/cuml/cuml/multiclass/multiclass.py

```diff
@@ -221,60 +135,129 @@ class OneVsRestClassifier(MulticlassClassifier):
         type. If None, the output type set at the module level
         (`cuml.global_settings.output_type`) will be used. See
         :ref:`output-data-type-configuration` for more info.
+    strategy: string {'ovr', 'ovo'}, default='ovr'
+        Multiclass classification strategy: 'ovr': one vs. rest or 'ovo': one
+        vs. one
+
+    Attributes
+    ----------
+    classes_ : float, shape (`n_classes_`)
+        Array of class labels.
+    n_classes_ : int
+        Number of classes.
+
+    Examples
+    --------
+    >>> from cuml.linear_model import LogisticRegression
+    >>> from cuml.multiclass import MulticlassClassifier
+    >>> from cuml.datasets.classification import make_classification
+
+    >>> X, y = make_classification(n_samples=10, n_features=6,
+    ...                            n_informative=4, n_classes=3,
+    ...                            random_state=137)
+
+    >>> cls = MulticlassClassifier(LogisticRegression(), strategy='ovo')
+    >>> cls.fit(X, y)
+    MulticlassClassifier(estimator=LogisticRegression())
+    >>> cls.predict(X)
+    array([1, 1, 0, 1, 1, 1, 2, 2, 1, 2])
     """
 
     def __init__(
-        self, estimator, *args, handle=None, verbose=False, output_type=None
+        self,
+        estimator,
+        *,
+        handle=None,
+        verbose=False,
+        output_type=None,
+        strategy="ovr",
     ):
+        warnings.warn(
+            "MulticlassClassifier was deprecated in version 25.12 and will be "
+            "removed in version 26.02. Please use OneVsOneClassifier or "
+            "OneVsRestClassifier directly instead.",
+            FutureWarning,
+        )
+
         super().__init__(
-            estimator,
-            *args,
-            handle=handle,
-            verbose=verbose,
-            output_type=output_type,
-            strategy="ovr",
+            estimator, handle=handle, verbose=verbose, output_type=output_type
         )
+        self.strategy = strategy
 
     @classmethod
     def _get_param_names(cls):
-        param_names = super()._get_param_names()
-        param_names.remove("strategy")
-        return param_names
+        return [*super()._get_param_names(), "strategy"]
 
 
-class OneVsOneClassifier(MulticlassClassifier):
+class OneVsRestClassifier(_BaseMulticlassClassifier):
     """
     Wrapper around Sckit-learn's class with the same name. The input can be
     any kind of cuML compatible array, and the output type follows cuML's
     output type configuration rules.
 
-    Berofe passing the data to scikit-learn, it is converted to host (numpy)
+    Before passing the data to scikit-learn, it is converted to host (numpy)
     array. Under the hood the data is partitioned for binary classification,
     and it is transformed back to the device by the cuML estimator. These
     copies back and forth the device and the host have some overhead. For more
     details see issue https://github.com/rapidsai/cuml/issues/2876.
```

Comment:
I find it somewhat odd to link to a GitHub issue here. 🤷🏻 Maybe we can clean up these docs in a follow-up?

Reply:
Agreed. I'd like us to add a cupy-native implementation (these estimators are pretty simple to implement), which would obviate the need for this doc at all.

I'll open a follow-up issue.